src/set.rs

```diff
@@ -0,0 +1,33 @@
+use crate::config::{get_config_toml, write_toml};
+use crate::error::PasswordStoreError;
+use clap::Values;
+use keyring::Keyring;
+use std::process::Command;
+use std::str::from_utf8;
+use toml::Value;
+
+fn get_random_password() -> Result<String, PasswordStoreError> {
+    let pass = Command::new("pwgen")
```

Comment:
Consider using an external crate to generate passwords. [passwords](https://crates.io/crates/passwords) is fairly similar to `pwgen`.